### ISSUE TITLE
Iterate over repositories to raise the limit over 100 enforced by Github

### ISSUE THREAD

**Issue by yizeng** [dblock/fue#5]
Works fine when the depth is 100, but a GraphQL error is thrown when depth is 1000.

> ruby 2.5.1p57 (2018-03-29 revision 63029) [x64-mingw32]

![depth-1000-exception](https://user-images.githubusercontent.com/1937586/44236474-3acd4c00-a201-11e8-806d-7fd8cc7562e3.png)

**Comment by dblock** [comment]
That has probably little to do with depth, but has to do with something the graphql API is returning. Can you try to debug it and see what data it's choking on in `graphql_error.rb`?

**Comment by dblock** [comment]
Error message fixed in https://github.com/dblock/fue/commit/4a0655687ace5ff2421e3d2f921fe5105b1e85bd, already fixed in Graphlient.

With the same command you now get a better error: `Requesting 1000 records on the `repositories` connection exceeds the `last` limit of 100 records.`, which clarifies what's going on. I updated the README to say that the max limit is 100. We can probably paginate over repositories to increase the limit, I renamed this issue to represent what we want to do.
